resources/certificate.rb

```diff
@@ -38,7 +38,7 @@
 property :ec_curve,   String, default: lazy { node['acme']['ec_curve'] }, equal_to: %w(prime256v1 secp384r1 secp521r1)
 
 # From https://letsencrypt.org/docs/profiles/
-property :profile,    String, default: 'classic', equal_to: %w(classic shortlived tlsclient tlsserver)
+property :profile,    String, default: nil, equal_to: [nil] + %w(classic shortlived tlsclient tlsserver)
```

Comment:
Shouldn't the type also change to allow `nil`?

```suggestion
property :profile,    [String, nil], default: nil, equal_to: [nil, 'classic', 'shortlived', 'tlsclient', 'tlsserver']
```

Reply:
Yes, you're right.